### PR TITLE
feat: add constants for standard schemas

### PR DESCRIPTION
**Pull request by xlgmokha**
# Why is this needed?

## What does this do?

This change adds constants for standard schemas that are mentioned in
[RFC-7643](https://www.rfc-editor.org/rfc/rfc7643.html) and [RFC-7644](https://www.rfc-editor.org/rfc/rfc7644.html)

Fixes https://github.com/xlgmokha/scim-kit/issues/55

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### CODE REVIEW

File: lib/scim/kit/v2/schemas.rb [hunk 1]
```diff
@@ -11,6 +11,7 @@ module Schemas
         ENTERPRISE_USER = "#{EXTENSION}:enterprise:2.0:User"
         GROUP = "#{CORE}:Group"
         RESOURCE_TYPE = "#{CORE}:ResourceType"
+        SCHEMA = "#{CORE}:Schema"
```
Comment (knguyendh) [inline lib/scim/kit/v2/schemas.rb:14]: Not sure what the purpose of this is? Couldn't find it specified in RFC 7644 and 7643

File: lib/scim/kit/v2/messages.rb [hunk 2]
```diff
@@ -5,9 +5,12 @@ module Kit
     module V2
       module Messages
         CORE = 'urn:ietf:params:scim:api:messages:2.0'
+        BULK_REQUEST = "#{CORE}:BulkRequest"
+        BULK_RESPONSE = "#{CORE}:BulkResponse"
```
Comment (knguyendh) [inline lib/scim/kit/v2/messages.rb:9]: Thanks for anticipating this possible future request

File: lib/scim/kit/v2/schemas.rb (hunk 1 above)
Comment (xlgmokha) [reply lib/scim/kit/v2/schemas.rb:14]: I found a reference to this in [RFC-7643, Section 7](https://www.rfc-editor.org/rfc/rfc7643.html#section-7).

Comment (knguyendh) [reply lib/scim/kit/v2/schemas.rb:14]: Nice catch. Thanks!!!

**Pull request merged**